examples/src/main/python/pi.py

```diff
@@ -27,12 +27,16 @@
 if __name__ == "__main__":
     """
         Usage: pi [partitions]
+        
+        Monte Carlo method is used to estimate Pi in the below example.
     """
     spark = SparkSession\
         .builder\
         .appName("PythonPi")\
         .getOrCreate()
-
+    
+    # If no arguments are passed(i.e. `len(sys.argv) < = 1` ) 
```

Comment:
This isn't visible to end users, so don't know if this matters. It's already documented in the usage. I am not sure this meaningfully helps.

Reply:
So I think we should expect folks to read the examples in addition to running them. That being said I don't think we need this comment specifically.

Reply:
This is actually the first example on the spark doc and I wanted to know how the `pi` calculation was done. There was no mention of what algorithm is used for it, so took me a while to figure out the Monte-Carlo estimator was used and the logic is randomly generating over 100000 points to finally estimate the Pi value.

Reply:
@srowen Added a line to the [index.md](https://github.com/apache/spark/pull/19632/commits/173fd29f20eab1ca8384413548deb5aac509d629) as well, I think it would help the curious ones know what algorithm is used and why an argument(i.e. 10) is passed.